.github/wordlist.txt

```diff
@@ -7,7 +7,7 @@ antirez
 api
 APIs
 ASYNC
-asyncronous
```

Comment:
I'm not sure what this file is doing, but as someone who maintains a spell-checking tool, the presence of this entry is a red flag.

I'm working from this report: https://github.com/check-spelling-sandbox/libvalkey/actions/runs/15354064163/attempts/1#summary-43209009081

Anyway, the last thing anyone wants is to put a misspelled word into a dictionary as it'll mean that a tool will not report that misspelled word.